docs/data-sources/security_policies.md

```diff
@@ -13,14 +13,27 @@ Security Policies are defined in the Twingate Admin Console and determine user a
 ## Example Usage
 
 ```terraform
-data "twingate_security_policies" "all" {}
+data "twingate_security_policies" "all" {
+  name = "<your security policy's name>"
+  #  name_regexp = "<regular expression of security policy name>"
+  #  name_contains = "<a string in the security policy name>"
+  #  name_exclude = "<your security policy's name to exclude>"
+  #  name_prefix = "<prefix of security policy name>"
+  #  name_suffix = "<suffix of security policy name>"
+}
 ```
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Optional
 
+- `name` (String) Returns only security policies that exactly match this name. If no options are passed it will return all security policies. Only one option can be used at a time.
+- `name_contains` (String) Match when the value exist in the name of the security policy.
+- `name_exclude` (String) Match when the value does not exist in the name of the security policy.
+- `name_prefix` (String) The name of the security policy must start with the value.
+- `name_regexp` (String) The regular expression match of the name of the security policy.
+- `name_suffix` (String) The name of the security policy must end with the value.
 - `security_policies` (Attributes List) (see [below for nested schema](#nestedatt--security_policies))
```

Comment:
The `security_policies ` attribute is not an input. This type needs to be updated so it will be moved to the "Read-Only" section in the docs

Reply:
fixed